lesson-4-metrics-logs-observability/README.md

```diff
@@ -0,0 +1,98 @@
+Microservices Perf\&Scale testing
+=================================
+
+What is handy to know:
+
+* What is percentile (and histogram and average)
+* Sequential vs. parallel
+* Throughput vs. latency
+* Horizontal vs. vertical scaling
+* Understand pod resource **requests** and **limits** (quality of service for testng vs. prod settings)
```

Comment:
a good reminder since this was taught on lesson 3

Reply:
Yep, seen it there, just wanted to remind.